apps/node-fastify/src/test/test-utils.ts

```diff
@@ -0,0 +1,50 @@
+import { OrbSync } from 'orb-sync-lib';
+
+/**
+ * Deletes test data from the database to ensure clean test state.
```

Comment:
Can remove the comment as it does not add much value - the method name and parameters are clear enough to be self-descriptive